components/sysutils/ntfsprogs/ntfsprogs.p5m

```diff
@@ -72,14 +71,10 @@ file path=usr/include/ntfs-3g/types.h
 file path=usr/include/ntfs-3g/unistr.h
 file path=usr/include/ntfs-3g/volume.h
 file path=usr/include/ntfs-3g/xattrs.h
-file path=usr/lib/libntfs-3g.a
-link path=usr/lib/libntfs-3g.so target=libntfs-3g.so.871.0.0
-link path=usr/lib/libntfs-3g.so.871 target=libntfs-3g.so.871.0.0
-file path=usr/lib/libntfs-3g.so.871.0.0
+link path=usr/lib/libntfs-3g.so target=libntfs-3g.so.881.0.0
+link path=usr/lib/libntfs-3g.so.881 target=libntfs-3g.so.881.0.0
+file path=usr/lib/libntfs-3g.so.881.0.0
 file path=usr/lib/pkgconfig/libntfs-3g.pc
-file files/fstyp path=usr/lib/fs/ntfs-3g/fstyp
```

Comment:
did you miss fstyp ?

Reply:
Links here supposed yo be useful.

Reply:
Yes, I overlooked it. But folder fs is history in this version.
Where to put it now?
file files/fstyp path=usr/bin/ntfs-3g.fstyp

Reply:
It was never delivered by ntfsprogs, it's illumos interface.

Reply:
Ok. I have to restore the usage of this folder.